locales/nb_NO.json

```diff
@@ -1,375 +1,379 @@
 {
-    "badRequest": "Feil i forespørselen. Prøv igjen",
-    "continueListening": "Fortsett å lytte",
-    "continueWatching": "Fortsett å se",
-    "home": "Hjem",
-    "incorrectUsernameOrPassword": "Feil brukernavn eller passord",
-    "login": {
-        "connect": "Koble til",
-        "changeUser": "Bytt bruker",
-        "changeServer": "Bytt server",
-        "manualLogin": "Manuell innlogging",
-        "serverNotFound": "Serveren ble ikke funnet",
-        "serverDashboard": "Serverdashboard",
-        "serverAddressRequired": "Serveradresse er påkrevd",
-        "serverAddressMustBeUrl": "Serveradressen må være en gyldig URL",
-        "serverAddress": "Serveradresse",
-        "serverVersionTooLow": "Serveren må være versjon 10.7.0 eller nyere",
-        "selectUser": "Velg en bruker",
-        "rememberMe": "Husk meg",
-        "loginAs": "Logg inn som {name}",
-        "addServer": "Legg til server",
-        "selectServer": "Velg server",
-        "login": "Logg inn"
+  "3DFormat": "3D-format",
```

Comment:
We leave all locale files apart from en-us unchanged. Other languages are translated on [Weblate](https://translate.jellyfin.org/projects/jellyfin/jellyfin-vue/)